.changeset/early-melons-thank.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Always return a cloned array from getCollection
```

Comment:
```suggestion
Updates `getCollection()` to always return a cloned array
```
We're not super strict about patch PRs, but in general, if we can start with a present tense verb like Adds/Removes/Fixes/Refactors/Deprecates etc. then this tends to create a helpful descriptive sentence for readers who are inspecting the changelog to know what has changed, and whether they need to do anything.

Suggestion above, but as you'll get used to me saying, check for accuracy/nuance since you know the change better than I do!

Reply:
Brilliant, thanks! That's just the sort of detail I need.